templates/zerver/help/mute-a-user.md

```diff
@@ -13,7 +13,8 @@ have the following effects:
 
 * All messages sent by muted users, including the name, profile
   picture, and message content, are hidden behind a **Click here to
-  reveal** banner.
+  reveal** banner which can be used to reveal such messages. It is also
+  possible to [re-hide revealed messages of muted users](/help/mute-a-user#re-hide-revealed-messages-of-muted-users).
```

Comment:
Maybe something like this, to keep it shorter:

All messages sent by muted users, including the name, profile  picture, and message content, are hidden behind a **Click here to  reveal** banner.  A revealed message can later be [re-hidden](/help/mute-a-user#re-hide-revealed-messages-of-muted-users).

Reply:
Made these changes.